### PR TITLE
[FIX] Highlighted commands are shown in errors again

### PR DESCRIPTION
**Pull request by jpelay**
**Description**

There's a bug right now in Hedy were the keywords in the errors are not being highlighted:

image

This is due to a syntax error in `additional.css`. Now is back to normal:

image

### CODE REVIEW

**Comment by Felienne** [comment]
Haha I just noticed this while working on something and I thought *I* had broken it :) Thanks for the fix!

**Comment by jpelay** [comment]
> Haha I just noticed this while working on something and I thought _I_ had broken it :) Thanks for the fix!

hahaha No worries, I'm trying to get the tests to work and then it'll be ready!

**Comment by jpelay** [comment]
The test were failing in the command line because of timing. I looked up for a while, but couldn't get it to work without the wait. I left a TODO comment to tackle this in the future.

File: tests/cypress/e2e/hedy_page/editor_box.cy.js [hunk 1]
```diff
@@ -26,13 +26,15 @@ describe('Test editor box functionality', () => {
     cy.visit(`${Cypress.env('hedy_page')}#default`);
     // click on textaread to get focus
     cy.get('#editor > .ace_scroller > .ace_content').click();
-    // empty textarea
+    // We wait until the editor is focused
+    // TODO: replace this wait. The editor takes a while to be focused
+    cy.wait(1000);
```
Comment (Felienne) [inline tests/cypress/e2e/hedy_page/editor_box.cy.js:31]:
```suggestion
    cy.wait(2500);
```

**Comment by Felienne** [comment]
Let's see if a higher time works!

**Comment by jpelay** [comment]
> Let's see if a higher time works!

Yes let's see! Hopefully it will! For me it also worked without waits in the command line using the option `--browser chrome` but I'm not sure if that's feasible on Github Actions

**Comment by Felienne** [comment]
> > Let's see if a higher time works!
> 
> Yes let's see! Hopefully it will! For me it also worked without waits in the command line using the option `--browser chrome` but I'm not sure if that's feasible on Github Actions

I don't know either, so I am not sure how we can move this forward at the moment

**Comment by jpelay** [comment]
> I don't know either, so I am not sure how we can move this forward at the moment

According to Cypress, yes we can, but since I'm working in the debugger at the moment, I'll tackle this on Monday! Do I set this as a draft in the meantime?

**Comment by jpelay** [comment]
The test that was failing was a flaky cypress test that I already opened an issue for in #4455.

**Review by Felienne — APPROVED** [review]
Yes, works again and now with passing tests!

image

Will also deploy this now cause it is a bit annoying that is is broken.

**Comment by jpelay** [comment]
Sigh, these test failed in https://github.com/hedyorg/hedy/actions/runs/6158212111 :c in #4471

**Comment by Felienne** [comment]
> Sigh, these test failed in https://github.com/hedyorg/hedy/actions/runs/6158212111 :c in #4471

Let's not worry about it all too much, we can discuss tonight!

**Pull request merged**